modules/rank-eval/src/test/resources/rest-api-spec/test/rank_eval/10_basic.yml

```diff
@@ -90,8 +90,8 @@
 "Mean Reciprocal Rank":
 
   - skip:
-      version: " - 6.99.99"
-      reason: the ranking evaluation feature is only available on 7.0
+      version: " - 6.2.99"
+      reason: bwc changes in 6.3
```

Comment:
is this correct? why skip rest tests? aren't we only changing where indices are set internally?

Reply:
I changed this previously because I changed the place where the indices strings for serialized/deserialized. However, I now think this isn't necessary since even though the location where the string array is writen changes, the overall ordering on the wire shouldn't change, thus there should be nothing breaking here. I removed the version skip changes in the last update.